timesketch/frontend-ng/src/components/Explore/TimelineComponent.vue

```diff
@@ -0,0 +1,581 @@
+<!--
+Copyright 2021 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!--
+  A generic timeline component that provides common timeline related
+  functionality and allows customization of the template. That way a timeline
+  can be represented as a chip or a table row.
+-->
+<template>
+  <span>
+    <v-dialog v-if="timelineStatus === 'processing'" v-model="dialogStatus" width="600">
```

Comment:
It should be fixed now. On the left side, there is currently no option to active the menu (because we do not have clickable table rows). Maybe we can add that in a later iteration.

Reply:
Sounds good. Thanks